CONTRIBUTING.md

```diff
@@ -166,14 +182,17 @@ For any new change, which involves changing any of the python SDK components and
 3. You should commit all your current set of changes for SDK and UX using `git commit`.
 4. Clean all untracked files using `git clean -fdx`
 5. Run `yarn install` and `yarn buildall` to build the UX changes.
-6. Run `pip install -e responsibleai` to install responsibleai locally.
-7. Run `pip install -e raiwidgets` to install raiwidgets locally.
+6. Run `pip install -e raiwidgets` to install raiwidgets locally.
+7. Run `pip install -e responsibleai` to install responsibleai locally.
```

Comment:
Does the order installing raiwidgets/responsibleai matter?

Reply:
Yes, it actually does!  You need to install raiwidgets first.  Otherwise, it will overwrite the dependencies.